src/test/resources/hw1/PlayAllTests.xml

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
+<suite name="All">
```

Comment:
Here you should not use declaration of classes, take a look for  `<suite-files>` tag

Reply:
Could you clarify here please? What exactly needs to be changed?